app/controllers/wpcc/your_results_controller.rb

```diff
@@ -53,5 +56,13 @@ def salary_message
         salary_frequency: session[:salary_frequency]
       )
     end
+
+    def default_salary_frequency
+      Hash[
+        Wpcc::YourDetailsForm::SALARY_FREQUENCIES.map do |elem|
+          [elem, elem == 'year' ? 'month' : elem]
```

Comment:
I found this hard to read at first because of the lack of parenthesis. Is it possible to improve the readability of this line of code, please?